packages/liveblocks-react-tiptap/src/mentions/MentionExtension.ts

```diff
@@ -110,86 +108,9 @@ const notifier = ({
   });
 };
 
-export const MentionExtension = Node.create<MentionExtensionOptions>({
-  name: LIVEBLOCKS_MENTION_TYPE,
-  group: "inline",
-  inline: true,
-  selectable: true,
-  atom: true,
-
+export const MentionExtension = Extension.create<MentionExtensionOptions>({
```

Comment:
Just so I understand, this was turned into an extension so you could use the node elsewhere?

Reply:
Yes! In our case, when we create the blocknote schema